docs/why.qmd

```diff
@@ -2,24 +2,313 @@
+ need for formatting SQL strings
+
+**Data scientists** can use Ibis to:
+
+- extract a sample of data for local iteration with a fast local backend
+- prototype with the same API that will be used in production
+- preprocess and feature engineer data before training a machine learning model
+
+### Ibis for data platforms
+
+Data platforms can use Ibis to quickly bring a fully-featured Python dataframe
+library with minimal effort to their platform. In addition to a great Python
+dataframe experience for their users, they also get integrations into the
+[broader Python and ML ecosystem](#ecosystem).
+
+Often, data platforms evolve to support Python in some sequence like:
 
-## Portability
+1. Develop a fast query engine with a SQL frontend
+2. Gain popularity and need to support Python for data science and ML use cases
+3. Develop a bespoke pandas or PySpark-like dataframe library and ML
+ integrations
 
-You can reuse the same code across different backends.
+This third step is where Ibis comes in. Instead of spending X money and Y time
+developing a bespoke Python dataframe library, you can create an Ibis backend
+for your data platform [in as little as four hours for an experienced Ibis
+developer](https://voltrondata.com/resources/new-ibis-backend-shipped-in-four-hours-druid)
+or, more typically, on the order of
+[one](https://github.com/ibis-project/ibis/pull/7954) or
+[two](https://github.com/ibis-project/ibis/pull/7303) months for a new
+contributor.
+
+::: {.callout-warning title="Why not the pandas or PySpark APIs?" collapse="true"}
+The pandas API inherently does not scale due to its single-threaded design,
+ordered index, and a lot of API baggage. The creator of pandas (and Ibis!) has
+[talked about the issues with pandas
+publicly](https://wesmckinney.com/blog/apache-arrow-pandas-internals/). While
+there have been projects attempting to scale the pandas API, they always result
+in a dubious support matrix. You can see that with
+[Modin](https://modin.readthedocs.io/en/stable/supported_apis/dataframe_supported.html)
+or [pandas on Spark (formerly known as
+Koalas)](https://spark.apache.org/docs/latest/api/python/user_guide/pandas_on_spark/supported_pandas_api.html).
+
+[Google BigQuery
+DataFrames](https://github.com/googleapis/python-bigquery-dataframes) are a more
+modern attempt to scale the pandas API, built on top of Ibis. If you are going
+to build a pandas API, we recommend you take a look at this project.
+
+PySpark is a great API for Spark, but not very Pythonic and tightly coupled to
+the Spark execution engine.
+
+Ibis takes inspiration from pandas and PySpark -- and R and SQL -- but is designed to be
+scalable from the start. If offers a neutral,
+[self-governed](https://github.com/ibis-project/governance) open source option
+for your data platform.
+:::
+
+### Ibis for data organizations
+
+Data organizations can use Ibis to standardize the interface for SQL and Python
+data practitioners. It also allows them to:
```

Comment:
The `them` here is unclear -- is it referring to the data practitioners or the data organizations?